projects/saturn/src/main/java/io/fairspace/saturn/config/Config.java

```diff
@@ -68,6 +68,7 @@ public static class Auth {
         public String clientId = "workspace-client";
         public boolean enableBasicAuth;
         public String superAdminUser = "organisation-admin";
+        public String[] defaultRoles = new String[0];
```

Comment:
wouldn't it be better to have it as set?
```
    @JsonSetter(nulls = Nulls.AS_EMPTY)
    public final Set<String> defaultRoles = new HashSet<>();
```

Reply:
I don't know, isn't a HashSet a bit of overkill for max 4 values? I would hash only bigger sets.

And also, will it serialize automatically? I don't like to write a custom serializer for this when this works OOTB.

Reply:
yes, it should work well, also it would made it consistent with other settings, e.g. features.
For `new String[0]` - have you checked what happens, when you define defaultRoles as null or empty? 
I'm not 100% sure, but I think we used to have some problems with properties defined this way...

Reply:
ok I'll change it!